front-end/src/components/Dashboard/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import styled from 'styled-components';
+import SideBarNav from "../../shared/SideBarNav";
+import jwt_decode from 'jwt-decode'
+
+
+
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            created_at: '',
+            id: '',
+            email: '',
+            password: '',
+            isLoaded: false
+        };
+    }
+        componentDidMount() {
+            let token =  localStorage.getItem('access_token');
+            let decoded = jwt_decode(token);
+            console.log({decoded});
+            return fetch("http://localhost:3000/users/" + decoded.id, {
+                method: 'GET',
+                headers:{
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json',
+                    'Authorization': 'Bearer '  + token ,
+                }
+
+            })
+                .then(res => res.json())
+                .then(
+                    (result) => {
+                        console.log(result[0]);
```

Comment:
Can we please remove this `console.log`?

Reply:
Removed